src/resources/i18n/translations/en.json

```diff
@@ -1,378 +1,378 @@
 {
-	"18 per year after that": "$18 per year after that",
+	"18 per year after that": "$18 yearly afterwards",
 	"A-Z": "A-Z",
-	"ARIA HINT - Contact button": "open your email client and email us.",
+	"ARIA HINT - Contact button": "open your e-mail program and send us an e-mail.",
```

Comment:
"E-mail" avoids a lo of errors in translation because the dash is not optional in many languages.
Adding it into a language that doesn't have it is far less of a problem than forgetting it.